test/e2e/positive.go

```diff
@@ -50,4 +71,98 @@ var _ = ginkgo.Describe("[PositiveFlow] Deployer validation", func() {
 			gomega.Expect(vo.Errors).To(gomega.BeEmpty())
 		})
 	})
+
+	ginkgo.Context("with cluster with all the deployed components", func() {
+
+		ginkgo.It("should have API components", func() {
+			mf, err := api.GetManifests()
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
+
+			key := getObjKey(mf.Crd.Name, mf.Crd.Namespace)
+			err = hp.WaitForObjectToBeCreated(key, &v1.CustomResourceDefinition{})
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
```

Comment:
this must be ensured by `deploy`, so it seems a bit redundant?

Reply:
Most of it, yes. I'll probably remove it